clients/src/main/java/org/apache/kafka/common/RecordReader.java

```diff
@@ -14,10 +14,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.apache.kafka.tools;
+package org.apache.kafka.common;
```

Comment:
Another possibility is to move this to the tools module under the package org.apache.kafka.tools.api, which seems a bit cleaner?

Reply:
This would follow a similar pattern as the storage api classes, which seems reasonable to me.

Reply:
@junrao @ijuma thanks for feedback.

> Another possibility is to move this to the tools module under the package org.apache.kafka.tools.api, which seems a bit cleaner?


tool module depends on core module, so I will add new subproject (tools-api) to tool moulds to avoid circular dependency.

Reply:
tools doesn't depend on core, only tool tests depend on core. Right?

Reply:
> tools doesn't depend on core, only tool tests depend on core. Right?

ok, will use new package instead.

Reply:
It's fine to use a separate module as you had.

Reply:
Yes, makes sense.